SU-SWS/saa_victoria_subtheme
# Adapt 884  logo files update

# READY FOR REVIEW

# Summary
- Updated logo files (brand bar and footer)
- Rebuilt theme to now include decanter template updates
- Updated readme

# Review By (Date)
- 10/21

# Urgency
- Medium - waiting for logo update

# Steps to Test

1. Pull down the branch or see this site: https://saauserguide-test.sites.stanford.edu/ 
2. Clear the site cache
3. Make sure the logos show in brand bar and footer (note that the update moves the 'alumni' text down a little to vertically align well with the bottom of the logo
before:
![image](https://user-images.githubusercontent.com/12898896/96510334-9731b900-121a-11eb-9972-874eae086d6a.png)

after:
![image](https://user-images.githubusercontent.com/12898896/96510306-8c772400-121a-11eb-82ae-48a95caae62f.png)

Also should be changed in footer, so check that 'alumni' is showing lower in the footer logo also.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket ADAPT-884
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
